### DISCUSSION TITLE
Use case eBay

### DISCUSSION THREAD

**Discussion by Solverz-0** [firefly-iii/firefly-iii#6537] [Firefly III] [answered]
Hoping for advise on a use case with eBay.

Let's say I sell something for £100 on eBay, money comes from eBay revenue account to my asset account.
Then after this I purchase £10 postage label to ship the parcel. So money moves from asset account to expense account.

Would this be the best way? Should I bother listing that expense from a specific budget or should I sold_value - postage_value and process the final value as income from revenue to asset?

**Comment by JC5** [comment, marked as answer]
I would put it under a specific budget (and its own "postal services" expense account). They are two distinct transactions after all. You may not need a budget for it, because if you don't sell anything, you don't spend the money.

**Reply by Solverz-0** [reply]
Ahh okay thank you, so I could put it under a category instead and maybe link it to the income of money from the sale?

**Reply by JC5** [reply]
Yep!
